### ISSUE TITLE
Make some named locations un-reservable.

### ISSUE THREAD

**Issue by MicahO** [CentralAZ/Rock-CentralAZ#126]
# Feature Request

**Describe the Feature Request**
With the release of Rock 9.0 and the Group Scheduler, the number of Named Locations will greatly increase. Many of these new locations are not reservable areas, and could introduce a large amount of clutter in the room scheduler.

**Describe Preferred Solution**
A checkbox on the individual room (or named location) that said "This Location Can Be Scheduled" would allow both systems to work together nicely.

**Describe Alternatives**
Michael Garrison suggested an alternative where rooms/locations/etc were a defined type and certain defined types could be scheduled. I see positives and negatives to both approaches but am not certain which would be better.

**Providing Code**
No.

**Comment by jrockman13** [comment]
I would agree with the defined type idea. I don't want any of my serving positions to be reserved....or my "containers".

**Comment by trfouger** [comment]
I would agree with the defined type idea as well.  I see where you turn these tickets around really quickly.  We are going live with Rock and this Room Management piece in November.  Any chance it would be ready that quickly?

**Comment by tcavaletto** [comment]
Add Property to reservation type of Reservation Location Types

**Comment by tcavaletto** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Make some named locations un-reservable.](https://trello.com/c/vIRULV5l/7-make-some-named-locations-un-reservable)

**Comment by milescarmany** [comment]
We have a use-case that seems to overlap with this request a bit. We have a Green Room back-stage we would like to keep hidden from most staff, but the Worship Team/Lead Pstor's Admin occasionally needs to request special resources for Easter, etc. that Facilities team is responsible for delivering.

Currently we have to omit this room from our locations to help herd the cats that view request systems like an order menu and handle any special needs for these rooms out-of-band which defeats the "all the same info in the same system" mentality we are trying to cultivate.
